### PR TITLE
Add page for November Hsinchu sprint

### PR DESCRIPTION
**Pull request by YenPeiChen07**
This PR is to add a page for the Hsinchu sprint on November 2nd.

### CODE REVIEW

**Comment by yungyuc** [comment]
@YenPeiChen07 , please tag those who should review and comment.

File: content/pages/sprint/2024/11-hsinchu.rst [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+Hacking Session
+------------------
+
+It aims to encourage collaboration and interaction among developers through project 
+participation. The projects cover various fields, including but not limited to science, 
+numerical computation, and engineering. You are also encouraged to share your own projects 
+in scisprint. Refer to `project list <#project-list>`__ below for more details.
+
+Project List
```
Comment (YenPeiChen07) [inline content/pages/sprint/2024/11-hsinchu.rst:67]: Project list need to be updated with another PR.

File: content/pages/index.rst [hunk 2]
```diff
@@ -8,15 +8,11 @@ home
 .. raw:: html
     
     <h2 class="text-2xl text-center mb-5">Upcoming Events</h2>
-    <div class="flex flex-row flex-wrap text-center">
-        <div class="md:w-1/2 w-full flex-col space-y-1 items-center">
-            <h3 class="text-lg">scisprint 2024 October in Hsinchu</h3>
+    <div class="flex flex-row flex-wrap text-center items-center justify-center">
```
Comment (YenPeiChen07) [inline content/pages/index.rst:11]: Make the single event align to center.
image

File: content/pages/sprint/2024/11-hsinchu.rst [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
++++++++++++++
+
+TBD
+
+Sign Up
```
Comment (YenPeiChen07) [inline content/pages/sprint/2024/11-hsinchu.rst:72]: Registration link need to be update with another PR.

**Comment by YenPeiChen07** [comment]
@meng5113, I created the sprint page for November Hsinchu sprint, please take a look. 

Since applying for parking coupon requires the event page, I would like to merge it first. If you find anything that need fixing, please feel free to comment on this PR.
